civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6312,11 +6312,18 @@ function civihr_employee_portal_hrreport_get_exposed_filter_default_values($view
   if (!empty($dateFilter['expose']['identifier'])) {
     $identifier = $dateFilter['expose']['identifier'];
 
-    $defaultFromDate = !empty($dateFilter['default_date']) ? $dateFilter['default_date'] : '';
-    $defaultToDate = !empty($dateFilter['default_to_date']) ? $dateFilter['default_to_date'] : '';
+    $defaultFromDate = !empty($dateFilter['default_date']) ? $dateFilter['default_date'] : null;
```

Comment:
sorry I just noticed this .. but I guess it would make more sense to orgnize the code like this :

```php
    if (empty($params[$identifier]['min'])) {
      $defaultFromDate = !empty($dateFilter['default_date']) ? $dateFilter['default_date'] : null;

      if ($defaultFromDate) {
        $from = new DateTime();
        $from->modify($defaultFromDate);
        $params[$identifier]['min'] = $from->format('Y-m-d');
      }
    }
```

the same applies for $params[$identifier]['max']